cypress/e2e/GUI/pagination_and_elements_pag5.cy.js

```diff
@@ -0,0 +1,36 @@
+/// <reference types="Cypress" />
+describe('Paginação', () => {
+    beforeEach(() => {
+      for (let page = 1; page <= 5; page++) {
+        cy.intercept(
+          'GET',
+          `/customers?page=${page}&limit=10&size=All`,
+          { fixture: `page${page}&limit10All.json` }
+        ).as(`pag${page}limit10all`);
+      }
+      cy.visit('/') 
+      cy.wait('@pag1limit10all');
+    });
+    context('Quando valido que estou na página 1', () => {
+        it('Então deve conter página 1 de 5', () => {
+            cy.get('[data-testid="pagination"] > span').then(($confirmElement) => {
+                if($confirmElement.length > 0) {
+                    cy.get('[data-testid="pagination"] > span')
```

Comment:
Que tal algo mais simples, como: `cy.contains('span', 'Page 1 of 5).should('be.visible')`?